docs/HowToDebugCrashWithAddr2lineTool.md

```diff
@@ -0,0 +1,163 @@
+# How to use Addr2Line tool for debugging
+**addr2line** is a tool which comes with the Toolchain in Linux.
+It converts the given address into file names and line numbers.  
+Developers can debug the TizenRT ELF/AXF output file through **addr2line**.
+It is especially helpful while debugging a crash and can be used to find the 
+exact line number where the crash occured and also the function callstack
+which resulted in the crash.
+
+## Contents
+- [Prerequisites](#prerequisites)  
+- [How to use addr2line](#how-to-use-addr2line)
+- [How to debug a crash with addr2line](#how-to-debug-a-crash-with-addr2line)
+- [Example debugging scenraio](#Example-debugging-scenario)
+
+## Prerequisites
+1. To use addr2line, please enable below config.  
+
+In menuconfig, enable **CONFIG_DEBUG_SYMBOLS**.
+
+```
+menuconfig -> Debug Options -> Generate Debug Symbols
+```
+
+2. Add your toolchain to the system path
```

Comment:
How about making a way to do this on dbuild? Because we already guide the docker instead of installing toolchain.